os/arch/arm/src/imxrt/imxrt_irq.c

```diff
@@ -442,8 +426,10 @@ void up_irqinitialize(void)
 #endif
 
 
-	irq_attach(IMXRT_IRQ_BUSFAULT, imxrt_busfault, NULL);
-	irq_attach(IMXRT_IRQ_USAGEFAULT, imxrt_usagefault, NULL);
+	irq_attach(IMXRT_IRQ_BUSFAULT, up_busfault, NULL);
+	up_enable_irq(IMXRT_IRQ_BUSFAULT);
+	irq_attach(IMXRT_IRQ_USAGEFAULT, up_usagefault, NULL);
+	up_enable_irq(IMXRT_IRQ_USAGEFAULT);
```

Comment:
Let us not enable these interrupts explicitly. 
These will be enabled at line 467 if CONFIG_SUPPRESS_INTERRUPTS is not defined. Else these interrupts will be treated as hardfault. This is expected behavior.

Reply:
Done.